deepforest/main.py

```diff
@@ -453,15 +454,22 @@ def predict_tile(self,
         self.model.eval()
         self.model.nms_thresh = self.config["nms_thresh"]
 
+        if raster_path is not None:
+            warnings.warn(
+                "The 'raster_path' argument is deprecated and will be removed in future versions. Use 'path' instead.",
+            DeprecationWarning
+            )
+            path = raster_path
+
         if (raster_path is None) and (image is None):
```

Comment:
This line also need to be changed from `raster_path` to `path`

Reply:
okay i removed extra spaces after the  `=` and changed from `raster_path` to `path`.

Reply:
@ethanwhite  please approve the testing workflow.